selfdrive/car/hyundai/hyundaican.py

```diff
@@ -67,21 +67,26 @@ def create_clu11(packer, frame, clu11, button):
   return packer.make_can_msg("CLU11", 0, values)
 
 
-def create_lfa_mfa(packer, frame, enabled):
+def create_lfahda_mfc(packer, enabled, hda_active=0, hda_set_speed=0):
   values = {
-    "ACTIVE": enabled,
+    # HDA_C_State:
```

Comment:
I think this kind of stuff should really live in the DBC. I could easily see it getting lost in a cleanup or refactor. What happens when this gets out of sync with the DBC defined comments/vals?

Reply:
yeah, i was just maintaining what was already there

Reply:
value definitions moved to https://github.com/commaai/opendbc/pull/338